Geometry/TrackerCommonData/data/PhaseI/pixfwdCylinder.xml

```diff
@@ -61,6 +61,14 @@ root volume plus an anchor point for positioning.
  <Constant name="CylindersServiceZ0"   value="0*mm"/>
  <Constant name="CylindersServiceZMin" value="[CylindersServiceZoff]"/>
  <Constant name="CylindersServiceZMax" value="[CylindersOuterLength]"/>
+
+ <Constant name="CarbonFiberSheetNearIPRmin" value="101.0*mm"/>
+ <Constant name="CarbonFiberSheetNearIPRmax" value="168.0*mm"/>
+ <Constant name="CarbonFiberSheetNearIPThickness" value="0.2*mm"/>
+
+ <Constant name="FlexCableDepth" value="0.90*mm"/>
```

Comment:
Ok, I do think there is a small error here; we factored in 28 cables per portcard block when in fact there are 14.  So the density of the FlexCable3 material should be reduced by a factor of 2.  We'll put this in the next request.